### PR TITLE
PORTALS-500

### PR DESCRIPTION
**Pull request by leem42**

### CODE REVIEW

File: src/lib/containers/QueryWrapperMenu.tsx [hunk 1]
```diff
@@ -72,10 +96,12 @@ export default class QueryWrapperMenu extends React.Component<QueryWrapperMenuPr
       changed
     */
     const { rgbIndex } = this.props
-    const hasPropsChanged = prevProps.rgbIndex !== rgbIndex
-    if (hasPropsChanged) {
+    if (rgbIndex !== prevProps.rgbIndex) {
```
Comment (jay-hodgson) [inline src/lib/containers/QueryWrapperMenu.tsx]: Think about comparing another property value (rather than color set).  Use key property??

**Comment by jay-hodgson** [comment]
Suggest changing accordion column layout from col-xs-2 to something like "col-xs-12 col-sm-3"

File: README.md [hunk 2]
```diff
@@ -180,82 +173,18 @@ Example 1: Rendering a Synapse Wiki page without any markdown pre-loaded
...
 ```jsx
-
   import {SynapseComponents} from 'synapse-react-client'
  
   <CustomMarkdownView>
     <SynapseComponents.Markdown 
       token={this.state.token}
-      ownerId={"syn14568473"}
-      wikiId={"582406"}
       markdown={"<wiki markdown that corresponds to syn14568473/582406>"}
```
Comment (leem42) [inline README.md]: change to #header and markdown stuff

File: README.md [hunk 3]
```diff
@@ -317,46 +246,10 @@ An example of a view with facets/stacked bar chart/table
...
-
-#### StaticQueryWrapper Props [**deprecated**]
-
-| Props  | Explanation |
-| ------------- | ------------- |
-| json  | Preloaded data from Synapse  |
-| token  | Session token to make authenticated requests to synapse  |
-| limit | The number of cards to show, e.g limit = 3 will only render 3 cards  |
-| sql  | SQL query to be run on Synapse  |
+| facetAliases: {} | Object with key = columnName and value as alias from query |
```
Comment (leem42) [inline README.md]: ?

File: src/__tests__/lib/containers/QueryWrapperMenu.test.tsx [hunk 4]
```diff
@@ -102,3 +101,66 @@ describe('it renders with basic functionality', () => {
...
+
+  it('renders with all the top level keys closed initially', async () => {
+    const { wrapper } = await createShallowComponent(props)
+    expect(wrapper.find('div.SRC-pointed-triangle-down')).toHaveLength(0)
+    expect(wrapper.find('div.SRC-accordion-key')).toHaveLength(2)
+  })
+
+  it('performs open, open, close', async () => {
+    const { wrapper } = await createShallowComponent(props)
+    const classIndicatingOpenAccordion = 'SRC-pointed-triangle-down'
```
Comment (leem42) [inline src/__tests__/lib/containers/QueryWrapperMenu.test.tsx]: make const between two files

File: src/lib/style/Portal.scss [hunk 5]
```diff
@@ -1237,17 +1230,79 @@ div.SRC-userImg {
...
+  justify-content: space-between;
+  padding-left: 10px;
+  span {
+    padding-right: 10px;
+  }
+}
+
+.SRC-accordion-key {
+  border: 1px solid #DCDCDC;
+  &.SRC-hoverWhiteText:hover {
```
Comment (leem42) [inline src/lib/style/Portal.scss]: might get removed

**Pull request merged**